From 4127d02f00561b458398ce2b5ced7ae853b23227 Mon Sep 17 00:00:00 2001
From: Bryan Quigley <bryan.quigley@canonical.com>
Date: Mon, 2 Jul 2018 16:48:21 -0400
Subject: [PATCH] [archive] fix stat typo

They're just missing the S_ in front of them so if that code gets
reached it fails.

Fixes: #1373
Resolves: #1374

Signed-off-by: Bryan Quigley <bryan.quigley@canonical.com>
Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/archive.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sos/archive.py b/sos/archive.py
index 263e3dd3f..fdf6f9a80 100644
--- a/sos/archive.py
+++ b/sos/archive.py
@@ -204,10 +204,10 @@ def _check_path(self, src, path_type, dest=None, force=False):
 
         def is_special(mode):
             return any([
-                stat.ISBLK(mode),
-                stat.ISCHR(mode),
-                stat.ISFIFO(mode),
-                stat.ISSOCK(mode)
+                stat.S_ISBLK(mode),
+                stat.S_ISCHR(mode),
+                stat.S_ISFIFO(mode),
+                stat.S_ISSOCK(mode)
             ])
 
         if force:
